src/components/itemsList.js

```diff
@@ -1,20 +1,46 @@
 /* eslint-disable react/button-has-type */
```

Comment:
- [ ] [OPTIONAL] It is best to fix your linter errors that use disabling comment. I kindly suggest that you fix this